### DISCUSSION TITLE
Model spawn without the need of floor area detection

### DISCUSSION THREAD

**Discussion by praddyum** [google/model-viewer#1930] [Q&A]
### Description

Previously when I tap on AR feature of my models inside model viewer they first asked to scan the ground and then only spawn on the ground but now they are spawned on its own. I am also attaching a live website to reproduce the bug. I don't know what I'am doing wrong.

#### Live Demo
https://praddy2009.github.io/TV/

### Version
https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js
- model-viewer: v1.3.0

### Browser Affected

- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [x] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

**Comment by elalish** [comment]
Yes, this is a UX improvement to ensure the user can at least see the model even if ARCore does not find the floor quickly. Once the floor is found, the model is placed on it. If you are using our default WebXR mode, then you can create a custom prompt for scanning the floor with HTML, see our [example](https://modelviewer.dev/examples/augmentedreality/#webXR).
